RDBM/Table.cpp

```diff
@@ -251,7 +253,7 @@ Table* Table::Set(unsigned int rowindex, std::string column, std::string value)
 		return nullptr;
 }
 
-Table& Table::AddTable(const Table& source)
+Table& Table::AddTable(const Table source)
```

Comment:
Why not reference?

Reply:
if you'll do something like 
table1.AddTable(table1);
it would cause infinite loop

Reply:
So we need additional check for this scenario, but not copying all tables. It's very expensive.